packages/pigeon/lib/java_generator.dart

```diff
@@ -200,5 +213,13 @@ void generateJava(JavaOptions options, Root root, StringSink sink) {
         _writeFlutterApi(indent, api);
       }
     }
+
+    indent.format('''private static HashMap wrapError(Exception exception) {
+\tHashMap<String, Object> errorMap = new HashMap<String, Object>();
+\terrorMap.put("${Keys.errorMessage}", exception.toString());
+\terrorMap.put("${Keys.errorCode}", null);
```

Comment:
This looks like an assertion violation (`PlatformException` cannot have a null `code`).